mne/channels/layout.py

```diff
@@ -445,10 +445,10 @@ def find_layout(info, ch_type=None, exclude='bads'):
         layout_name = 'magnesWH3600'
     elif has_CTF_grad:
         layout_name = 'CTF-275'
-    elif n_kit_grads <= 157:
-        layout_name = 'KIT-157'
-    elif n_kit_grads > 157:
-        layout_name = 'KIT-AD'
+    elif n_kit_grads > 0:
+        layout_name = _find_kit_system(info, n_kit_grads)
+        if layout_name == 'KIT-UMD':
+            return _auto_layout(info)
```

Comment:
Ok, I generated an automatic layout and then I nudged the rectangles around manually to minimize overlap, which seemed preferable for plot_topo. I noticed though that topomap also uses the same layout where a deterministic arrangement seems preferable?

![kit1](https://cloud.githubusercontent.com/assets/145771/16542741/1ad3cf14-4083-11e6-9032-d4062ae4a373.png)
![kit4](https://cloud.githubusercontent.com/assets/145771/16542742/1d97e64a-4083-11e6-9d22-12b3d08e1176.png)